docs/2.0/docs/pipelines/installation/addingexistinggitlabrepo.mdx

```diff
@@ -0,0 +1,879 @@
+# Bootstrap Pipelines in an Existing GitLab Project
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import PersistentCheckbox from '/src/components/PersistentCheckbox';
+import CustomizableValue from '/src/components/CustomizableValue';
+
+This guide provides comprehensive instructions for integrating [Gruntwork Pipelines](https://gruntwork.io/products/pipelines/) into an existing GitLab project with Infrastructure as Code (IaC). This is designed for Gruntwork customers who want to add Pipelines to their current infrastructure projects for streamlined CI/CD management.
+
+To configure Gruntwork Pipelines in an existing GitLab project, complete the following steps (which are explained in detail below):
+
+1. **(If using a self-hosted GitLab instance) Ensure OIDC configuration and JWKS are publicly accessible.**
+2. **Plan your Pipelines setup** by identifying all environments and cloud accounts/subscriptions you need to manage.
+3. **Bootstrap core infrastructure** in accounts/subscriptions that don't already have the required OIDC and state management resources.
+4. **Configure SCM access** using [machine users](/2.0/docs/pipelines/installation/viamachineusers) with appropriate Personal Access Tokens (PATs).
+5. **Create `.gruntwork` HCL configurations** to tell Pipelines how to authenticate and organize your environments.
+6. **Create `.gitlab-ci.yml`** to configure your GitLab CI/CD pipeline.
+7. **Commit and push** your changes to activate Pipelines.
+
+## Ensure OIDC configuration and JWKS are publicly accessible
```

Comment:
It's a bit odd to have this headline come BEFORE the next headline of "prerequisites" - feels like this belongs under that headline?